pmoves/services/github-runner-ctl/nats_publisher.py

```
@@ -0,0 +1,247 @@
+            envelope = create_event_envelope(
+                topic=subject,
+                payload=payload,
+                source=source,
+                correlation_id=correlation_id,
+                parent_id=parent_id,
+            )
+            await self._nc.publish(subject, json.dumps(envelope).encode())
+            NATS_EVENTS_PUBLISHED.labels(subject=subject, status="success").inc()
+            logger.debug(f"Published {subject}: {envelope['id']}")
+            return True
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)